components/busola-migrator/README.md

```diff
@@ -36,6 +36,7 @@ Before running the program, make sure your active Kubeconfig is pointing at the
 | **OVERRIDE_OIDC_CLIENT_ID** | Optional override for the OIDC client ID | None |
 | **OVERRIDE_OIDC_SCOPE** | Optional override for the OIDC scope | None |
 | **OVERRIDE_OIDC_USE_PKCE** | Optional override for the use of PKCE in OIDC  | None |
-| **APP_UAA_URL** | User Account and Authentication (UAA) server URL | None |
+| **APP_UAA_ENABLED** | Enable User Account and Authentication (UAA) migration functionality | `true` |
```

Comment:
Since all the other variables' descriptions take form of a noun description, and don't start with a verb, I'd suggest aligning with them to make it uniform and coherent (even though it's longer this way 😞 ):

```suggestion
| **APP_UAA_ENABLED** | Parameter specifying whether the User Account and Authentication (UAA) migration functionality is enabled | `true` |
```